homeassistant/helpers/config_validation.py

```diff
@@ -267,6 +297,19 @@ def url(value: Any) -> str:
     raise vol.Invalid('invalid url')
 
 
+def email(value: Any) -> str:
```

Comment:
Any specific reason `vol.Email` is not good enough?

(maybe just that voluptuous documentation is very bad! [here](https://github.com/alecthomas/voluptuous/blob/b052498611869cf216b6c3b40a7d07b9f236f56f/voluptuous/validators.py#L349))